src/main/resources/pan_ddm_carare.xsl

```diff
@@ -547,12 +545,13 @@
                 </isShownAt>
 
                 <!-- rights -->
+                <xsl:variable name="license" select="//bagmetadata:bagmetadata/ddm:DDM/ddm:dcmiMetadata/dcterms:license[@xsi:type=&apos;dcterms:URI&apos;]"/>
```

Comment:
It is then always an open access license here?

Reply:
it defaults to the same license as the dataset